infra/aws/e2e/accounts/attach.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+set -x
+
+for s in a b c d; do
```

Comment:
I don't understand how `a b c d` here is used. Can you add comments to explain?

Also, copyright info.